restore_ports/restore_ports.tcl

```diff
@@ -164,7 +170,7 @@ proc install_ports {operationList} {
         if {[catch {set res [mportlookup $name]} result]} {
             global errorInfo
             ui_debug "$errorInfo"
-            return -code error "lookup of portname $name failed: $result"
+            return -code error "lookup of portname $name failed: \"$result\""
```

Comment:
Do you expect issues with trailing whitespace or why do we need these quotes?

Reply:
It's been awhile since I created this PR so I forget why exactly I added them in the first place, but I think it was because I was having trouble telling what exactly the output was saying; they're more just grouping characters for human readers to make it easier to tell which part exactly is the variable

Reply:
@raimue do you want me to remove the quotes?